plugins/hls-fourmolu-plugin/hls-fourmolu-plugin.cabal

```diff
@@ -22,7 +22,7 @@ library
   build-depends:
     , base            >=4.12 && <5
     , filepath
-    , fourmolu        ^>=0.3
+    , fourmolu        ^>=0.3 || ^>=0.4
```

Comment:
Ideally, I'd remove 0.3 entirely, as 0.4 includes some important bug fixes. But this isn't possible while `stylish-haskell` is tied to older versions of GHC and Cabal.

(Same applies to Ormolu here - right now, the latest version will never be chosen by the dependency solver)

Reply:
Actually, looks like this only applies to GHC < 9. The `stylish-haskell` plugin is disabled for 9.0.

Reply:
Maybe adding a comment about, to remove 0.3 some day?

Reply:
> Maybe adding a comment about, to remove 0.3 some day?

Actually, I think there's no need. I'll probably end up at least trying to remove it every time I bump the upper bound, and eventually it'll be possible.